cmd/podman/build.go

```diff
@@ -57,22 +57,30 @@ func init() {
 
 	budFlags := buildahcli.GetBudFlags(&budFlagsValues)
 	flag := budFlags.Lookup("pull")
-	flag.Value.Set("true")
+	if err := flag.Value.Set("true"); err != nil {
+		logrus.Error("unable to set pull flag to true")
+	}
 	flag.DefValue = "true"
 	layerFlags := buildahcli.GetLayerFlags(&layerValues)
 	flag = layerFlags.Lookup("layers")
-	flag.Value.Set(useLayers())
+	if err := flag.Value.Set(useLayers()); err != nil {
+		logrus.Error("unable to set uselayers")
+	}
 	flag.DefValue = useLayers()
 	flag = layerFlags.Lookup("force-rm")
-	flag.Value.Set("true")
+	if err := flag.Value.Set("true"); err != nil {
+		logrus.Error("unable to set force-rm flag to true")
+	}
 	flag.DefValue = "true"
 
 	fromAndBugFlags := buildahcli.GetFromAndBudFlags(&fromAndBudValues, &userNSValues, &namespaceValues)
 
 	flags.AddFlagSet(&budFlags)
 	flags.AddFlagSet(&layerFlags)
 	flags.AddFlagSet(&fromAndBugFlags)
-	flags.MarkHidden("signature-policy")
```

Comment:
How about you make a helper function that takes flags, and "signature-policy" as options.
markHiddent(flags, "signature-policy")

Then change all calls to MarkHidden to use it.

Reply:
Buildah panics here. The intention behind that is to reveal such issues early as they could silently be introduced by a PR.

Reply:
Given we do similar for remoteclient, im inclined to do this as recommended.  ty